src/it/resources/org/checkstyle/suppressionxpathfilter/descendanttoken/InputXpathDescendantTokenNestedSwitch.java

```diff
@@ -0,0 +1,21 @@
+package org.checkstyle.suppressionxpathfilter.descendanttoken;
+
+public class InputXpathDescendantTokenNestedSwitch {
+
+void testMethod1() {
+        int x = 1;
+        int y = 2;
+        switch (x) {
+            case 1:
+                System.out.println("xyz");
+                break;
+            case 2:
+        switch (y) { // warn
```

Comment:
please fix formatting of this Input.
this is very weird on why it is not placing violation on it.

can you please debug this test and go inside Check to see why it is not placing violation?
lets consider it as defect and we need to debug it to fix it.